_rules/autocomplete-valid-73f2c2.md

```diff
@@ -279,3 +271,11 @@ Non-widget element that does not participate in sequential focus navigation.
 ```html
 <input type="button" role="none" tabindex="-2" autocomplete="username" />
 ```
+
+#### Inapplicable example 11
```

Comment:
Actually, need uppercase E at *E*xample, as you probably have figured out with the tests :-D